pkg/scheduler/schedule_one.go

```diff
@@ -718,7 +733,7 @@ func prioritizeNodes(
 	logger := klog.FromContext(ctx)
 	// If no priority configs are provided, then all nodes will have a score of one.
 	// This is required to generate the priority list in the required format
-	if len(extenders) == 0 && !fwk.HasScorePlugins() {
+	if len(sched.Extenders) == 0 && !sched.hasScoring(fwk) {
```

Comment:
Shouldn't it be just this now?

```suggestion
	if !sched.hasScoring(fwk) {
```

Reply:
Fixed